tests/test_inputs.py

```diff
@@ -55,6 +55,26 @@ def test_input_pulled_up():
     with pytest.raises(PinFixedPull):
         InputDevice(2, pull_up=False)
 
+def test_input_is_active_low_externally_pulled_up():
+    pin = Device.pin_factory.pin(4)
+    device = InputDevice(4, pull_up=None, active_state=False)
+    pin.drive_high()
+    assert repr(device) == '<gpiozero.InputDevice object on pin GPIO4, pull_up=None, is_active=False>'
+    assert not device.is_active
+    pin.drive_low()
+    assert repr(device) == '<gpiozero.InputDevice object on pin GPIO4, pull_up=None, is_active=True>'
+    assert device.is_active
+
+def test_input_is_active_low_externally_pulled_down():
```

Comment:
Was this supposed to be `test_input_is_active_high_externally_pulled_down` ?

Reply:
Fixed in c64aab36afa33c260980a2776f538bfdcdf636f2